chart/values.yaml

```diff
@@ -81,6 +81,7 @@ controller:
   disruptionCronEnabled: true
   disruptionRolloutEnabled: false
   disruptionDeletionTimeout: 15m # The duration after which a disruption will be marked as "stuck on removal" if its removal process exceeds this duration.
+  aggregateToClusterRole: false
```

Comment:
As an end user perspective - would it be possible to place a comment line here what this does when enabled and/or if it's required to run chaos-controller? 

Thanks :)